### ISSUE TITLE
After InnerTune 0.5.4 backup restoration, OuterTune keeps removing Liked/Added to Library songs

### ISSUE THREAD

**Issue by Username1-a** [OuterTune/OuterTune#60]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/dd3boh/OuterTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

1. Restore a backup from InnerTune version ≥0.5.4
2. Open the Songs tab, you will see the list of all the Liked/Added 5o Library songs restored from IT

### Expected behavior

Restored Liked songs should persist in OuterTune

### Actual behavior

3. After a couple of seconds, OuterTune will erase all those songs so you will end up with empty list of Liked/Added to Library songs

### Screenshots/Screen recordings

![Screenshot_20240930-213240_OuterTune](https://github.com/user-attachments/assets/07d6e406-d65f-4509-a2a4-5727a087ba2d)
![Screenshot_20240930-213312_OuterTune](https://github.com/user-attachments/assets/b29c92e9-7b55-4988-8217-4db8fcf9e418)
![Screenshot_20240930-213332_OuterTune](https://github.com/user-attachments/assets/722cdff3-80e3-44c7-8d0c-bfc27e25ef5b)

### Logs

_

### OuterTune version

0.6.2

### Android version

Android 13 QPR3

**Comment by mikooomich** [comment]
Do you happen to be signed into your youtube music account?

**Comment by Username1-a** [comment]
@mikooomich Yeah, I'm logged in (in InnerTune where the backup was made), if that's the case

Looks like even with clean OuterTune, it logs into google account as soon I restore a backup from InnerTune and I can't seem to just log off from google account in both InnerTune/OuterTune (can't see any button or anything like that)

**Comment by mikooomich** [comment]
Yeah it seems like the ytm sync is removing all the local likes then. I'll probably make the migration Create a playlist with old likes or something.

What you can try for the time being is restore the backup fully disconnected from network, then selecting all and adding all to a new playlist

**Comment by ancientPrinter** [comment]
Same issue here. So far, I've managed to avoid this by disabling the sync option right after reatoring the backup. I don't know how to sync with YT without losing the songs, though.

**Comment by thepatientwait** [comment]
Looks like this is a deliberate feature?

https://github.com/DD3Boh/OuterTune/blob/e00a083936b5f984de21647f02ff6640ed3677fe/app/src/main/java/com/dd3boh/outertune/utils/SyncUtils.kt#L89-L101

This leads to some undesirable behaviour where locally liked songs/albums are instantly removed from the library (at least when I tested just now).

This is just my opinion as a user but I feel like local likes should be pushed to YTMusic if the sync functionality is enabled.
Or maybe there should be a second option for local -> remote syncing if that is undesirable for some.

Not super familiar with this code base though, so maybe these changes could break something or maybe the API wouldn't allow liking from OuterTune idk
